CONTRIBUTING.md

```diff
@@ -71,12 +73,14 @@ Automated tests can be run with the command `yarn test`.
 If you're hacking on a component and want to test it in the documentation site
 follow the following steps.
 
-1. [Set up Jekyll to run the docs site locally](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll),
-if it's not already set up.
+1. Ensure that you have the Bundler Ruby gem installed
+   by running `gem install bundler`.
+   ([more info](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll#requirements)).
 1. Check out a working branch.
 1. Run `gulp docs`.
 1. Go into the documentation directory with `cd docs`.
-1. Run `bundle exec jekyll serve` and visit http://127.0.0.1:4000.
+1. Run `bundle install` if you haven't before.
```

Comment:
Still missing a mention of running `bundle update` periodically after the initial install.

Reply:
ARGH had this addition unsaved in a tab :(

Reply:
https://github.com/cfpb/capital-framework/pull/910/commits/5efd96cb27e052cea18fecc726f07932167b7b6f